.github/BOTMETA.yml

```diff
@@ -223,6 +223,8 @@ files:
   $lookups/redis.py:
     maintainers: $team_ansible_core jpmens
   $lookups/shelvefile.py: {}
+  $lookups/revbitspss.py:
```

Comment:
Can you move this entry before the previous one? `rev` comes after `red`, but before `she` in lexicographic order.